immuno_graphical/code/params.R

```diff
@@ -2,11 +2,65 @@ library(here)
 library(stringr)
```

Comment:
@KenLi93 can you confirm that all these changes are correct and needed in this PR?

Reply:
Hi David, yes these commits are all necessary for a correct build.

Thanks,
Kendrick